src/functionalTest/resources/scenarios/RIA-5953-change-direction-due-date-AboutToStart-no-directions-exist.json

```diff
@@ -0,0 +1,22 @@
+{
+  "description": "Change bail direction due date event - about to start no directions exist error",
```

Comment:
Can you write a scenario where there are at least 2 directions, and that the expectation should be the handler (ChangeDirectionDueDatePreparer) returns correct names and order?

Reply:
Thanks Thomas, added in this test case to the latest commit.